configure.ac

```diff
@@ -5158,6 +5158,7 @@ then
 	plugin_entropy="yes"
 	plugin_fscache="yes"
 	plugin_interface="yes"
+	plugin_ipc="yes"
```

Comment:
As this plugin seems to also work in AIX, would you like to add it below (around line 5200) to build it by default on this platform ?